misk/web/@misk/common/webpack.config.js

```diff
@@ -7,7 +7,7 @@ module.exports = {
   name: "library",
   mode: "production",
   entry: {
-    common: path.resolve(__dirname, "src/index.ts")
+    common: path.resolve(__dirname, "src/index.ts"),
```

Comment:
Not for this PR, but if we're using a `js` linter anywhere else, we could run it through our config files to keep them consistent?

Reply:
Good idea, I'll update the linting filters so the config files get picked up 👍